### ISSUE TITLE
API Authentication Failure (even though api keys work when used outside linode plugin)

### ISSUE THREAD

**Issue by programingnotes** [displague/terraform-provider-linode#20]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
```
// ♥ terraform -v
Terraform v0.11.8
+ provider.linode (unversioned)
```

### Affected Resource(s)
- linode\_instance

### Terraform Configuration Files
```hcl
provider "linode" {
  key = "key-as-plain-text-string"
}

resource "linode_linode" "terraform-example" {
  image = "Ubuntu 16.04 LTS"
  kernel = "Grub 2"
  name = "linode-example"
  group = "terraform-test"
  region = "Atlanta, GA, USA"
  size = 1024
  ssh_key = "your-ssh-id_rsa.pub-here"
  root_password = "your-server-password-here"
}
```

### Debug Output
**[link to the debug output Gist](https://gist.github.com/programingnotes/f0c9ac78d3dcdda9d9661757ae1e61c6).**

### Panic Output
- Terraform did not produce a panic

### Expected Behavior
What should have happened?
- expected authentication to be successful and an execution plan to be generated. 

### Actual Behavior
- Authentication Failed 

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `go get github.com/hashicorp/terraform`
2. `go get github.com/LinodeContent/terraform-provider-linode`
3.  `cd $GOPATH/src/github.com/LinodeContent/terraform-provider-linode/bin/terraform-provider-linode`
4. `go build -o terraform-provider-linode`
5. `mv $GOPATH/src/github.com/LinodeContent/terraform-provider-linode/bin/terraform-provider-linode/terraform-provider-linode $GOPATH/bin`
6. `mv$GOPATH/src/github.com/LinodeContent/terraform-provider-linode/linode-template.tf $GOPATH/bin`
7. `go get github.com/hashicorp/terraform`
8. `wget https://github.com/linode/docs-scripts/raw/master/hosted_scripts/terraform-linode-plugin/terraform-provider-linode`
9. `mv terraform-provider-linode ~/go_projects/bin/`
10. `chmod 750 $GOPATH/bin/terraform-provider-linode`
11. configure main.tf, initialize terraform, and attempt to generate plan by running `terraform plan`

### Important Factoids
- Basically followed steps outlined [here](https://www.linode.com/docs/applications/configuration-management/how-to-build-your-infrastructure-using-terraform-and-linode/)

### References
None, that I have seen yet

**Comment by displague** [comment]
Hi @programingnotes,

The steps and errors reported in this issue do not reflect the state of this repository  (this fork is progressing towards becoming an official provider and has taken substantial deviations from the original btobolaski repo and the LinodeContent fork).  

This repository is a work-in-progress, with two fairly large open issues that will bring further changes to the `linode_instance` resource (note that your code above references the `linode_linode` resource which no longer exists in this project).

You can try this repo out, none the less, using the instructions provided here: 
- https://github.com/displague/terraform-provider-linode#building-the-provider

Examples are provided here:
- https://github.com/displague/terraform-provider-linode/tree/master/examples

**Comment by programingnotes** [comment]
Hi @displague Thank you for the response and accept my apologies for opening up issue on wrong repo.  Will use the link suggested in your reply.
